aeron-driver/src/main/c/agent/aeron_driver_agent.h

```diff
@@ -164,6 +166,19 @@ typedef struct aeron_driver_agent_flow_control_receiver_change_log_header_stct
 }
 aeron_driver_agent_flow_control_receiver_change_log_header_t;
 
+typedef struct aeron_driver_agent_send_nak_message_header_stct
+{
+    int64_t time_ns;
+    int32_t session_id;
+    int32_t stream_id;
+    int32_t term_id;
+    int32_t term_offset;
+    int32_t nak_length;
+    int32_t address_length;
```

Comment:
To keep the structure simpler, I would just use a `struct sockaddr_storage` here instead of tracking the length to handle IPv4 and IPv6.